examples/experimental/i3s-17-and-debug/components/debug-panel.js

```diff
@@ -159,9 +159,9 @@ export default class DebugPanel extends PureComponent {
     });
   }
 
-  _onToggleUvChecker() {
-    const {useUvChecker} = this.state;
-    this.setState({useUvChecker: !useUvChecker}, () => {
+  _onToggleUvDebugTexture() {
+    const {showUVDebugTexture} = this.state;
+    this.setState({showUVDebugTexture: !showUVDebugTexture}, () => {
```

Comment:
Here we could go through the tiles and replace the textures. It may require us to import Texture contructor from luma.gl and actually create textures.

Reply:
We can't do it on loaders level. `Tile3D` class doesn't have `gl` context.

Reply:
True, but this file is not in the loader. It is in the app.